ldap/servers/slapd/back-ldbm/ldbm_config.c

```diff
@@ -1254,7 +1254,9 @@ ldbm_config_search_entry_callback(Slapi_PBlock *pb __attribute__((unused)),
             if (attrs) {
                 for (size_t i = 0; attrs[i]; i++) {
                     if (ldbm_config_moved_attr(attrs[i])) {
-                        slapi_pblock_set(pb, SLAPI_RESULT_TEXT, "at least one required attribute has been moved to the DB scecific configuration entry");
+                        /* operation_done() free's result text, so we must pass a copy */
```

Comment:
But slapi_pblock_set_pb_result_text already perform a strdup ...

Reply:
> But slapi_pblock_set_pb_result_text already perform a strdup ...

Ahh, you are confusing "pb_result_text" with "result_text"  :-)  Two different things